docs/quota_limit.md

```diff
@@ -2,35 +2,35 @@
 
 ## Training
 
-|                                    | Generic IC                      | Generic OD                      |
-| ---------------------------------- | ------------------------------- | ------------------------------- |
-| Max # training hours*              | 336 (14 compute days)           | 1344 (56 compute days)          |
-| Max # training images              | 1,000,000                       | 200,000                         |
-| Min # training images per category | 2                               | 2                               |
-| Max # tags per image               | multiclass: 1                   | NA                              |
-| Max # regions per image            | NA                              | 1,000                           |
-| Max # categories                   | 2,000                           | 1,000                           |
-| Min # categories                   | 2                               | 1                               |
-| Max image size                     | 20MB                            | 20MB                            |
-| Max image width/height             | 10,240                          | 10,240                          |
-| Min image width/height             | 10                              | 10                              |
-| Available regions                  | West US 2, East US, West Europe | West US 2, East US, West Europe |
-| Accepted image types               | jpg, png, bmp, gif, jpeg        | jpg, png, bmp, gif, jpeg        |
+|                                    | Generic IC                      | Generic OD                      | Product Recognition             |
+| ---------------------------------- | ------------------------------- | ------------------------------- | ------------------------------- |
+| Max # training hours*              | 336 (14 compute days)           | 1344 (56 compute days)          | 336 (14 compute days)           |
+| Max # training images              | 1,000,000                       | 200,000                         | 1,000,000                       |
```

Comment:
is product reco training with OD dataset?

Reply:
If so, 1M images could lead to like 10M images in the real IC training I suppose?

Reply:
Updated